gp-templates/profile-view.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
Can you explain the purpose of this file? If it doesn't get used by GlotPress it should probably be part of the plugin.